frontend/src/types/NMPFileManureStorageSystem.ts

```diff
@@ -11,19 +11,41 @@ export type ManureInSystem =
       data: NMPFileImportedManureData;
     };
 
+export type StorageShapes = 'Rectangular' | 'Circular' | 'SlopedWallRectangular';
+
 export type ManureStorage = {
   id: number;
   name: string;
   isStructureCovered: boolean;
   uncoveredAreaSqFt?: number;
+  SelectedStorageStructureShape: StorageShapes;
```

Comment:
Just a heads up: I'm planning to split these out into solid and liquid props in my change

And also make all these variable names camelcase